lib/tasks/locale.rake

```diff
@@ -289,7 +302,7 @@ namespace :locale do
     end
   end
 
-  desc "Convert PO files from all plugins to JS files"
+  desc "Convert PO files from all plugins to JS files(runnable from app namespace)"
```

Comment:
I don't think we need this extra comment, there are a lot of methods that are available in the app namespace (specifically _all_ of these are available when run from an engine)